components/series/SeriesTable.vue

```diff
@@ -422,6 +434,7 @@ export default class SeriesTable extends mixins(PrefixMixin) {
         buyHistory: axisLize(
           Object.assign({}, defaultBuyEvents, buyEvents[e.id] || {})
         ),
+        emoteCount: e.emoteCount,
```

Comment:
do you by any chance now it this will always be set? meaning that if no emotes have been applied, is 0 from subqsuid? otherwise we should do something like this 
```suggestion
        emoteCount: e.emoteCount || 0,
```
otherwise `<template>`will crash

Reply:
yeah you're right, no emotes returns an empty array

Reply:
thank you !